plugins/inputs/opcua_listener/subscribe_client.go

```diff
@@ -32,6 +32,52 @@ type SubscribeClient struct {
 	processingCancel context.CancelFunc
 }
 
+func CheckDataChangeFilterParameters(params *input.DataChangeFilter) error {
```

Comment:
Please do not export this function! It should not be used outside of this plugin!
```suggestion
func checkDataChangeFilterParameters(params *input.DataChangeFilter) error {
```